app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+###
+#  User class for authenticating via WebAuth
+###
+class User < ActiveRecord::Base
```

Comment:
Just out of curiousity, is there a requirement to store these users?  I'm not familiar enough w/ the whole app to know we'll need user records persisted in the DB or not. If not we could probably get away with a plain ruby object that doesn't require any persistence.

If you know that you def. need to persist user data and associate user records with other objects, then 👍